OpenShift/init.sh

```diff
@@ -41,30 +41,28 @@ echo
 echo "Configuring project permissions..."
 echo "=================================="
 echo
-# Grant Default CI Account Edit Access to All Projects and OpenShift Project
+# Grant Default Nexus Account Edit Access to All Projects and OpenShift Project
 oc policy add-role-to-user edit system:serviceaccount:${OPENSHIFT_NEXUS_PROJECT}:default -n ${OPENSHIFT_NEXUS_PROJECT}
```

Comment:
I understand this script is for test, but why do you need edit access from that service account?